frontend/src/features/admin-form/responses/ResponsesPage/email/EmailResponsesTab.tsx

```diff
@@ -30,12 +30,7 @@ export const EmailResponsesTab = (): JSX.Element => {
           </Text>
         </Skeleton>
         <Text textStyle="body-1">
-          FormSG does not store responses in Email mode. To collate the
-          responses in your Outlook Inbox, use the{' '}
-          <Link isExternal href={OGP_FORMSG_COLLATE}>
```

Comment:
In lines 4 and 5, it seems that the `Link` and `OGP_FORMSG_COLLATE` imports are no longer used. perhaps we can delete the imports and the `OGP_FORMSG_COLLATE` link in `links.ts` file

Reply:
e9bcfd4e2